packages/clay-label/src/index.tsx

```diff
@@ -67,6 +73,12 @@ const ClayLabel = React.forwardRef<HTMLAnchorElement | HTMLSpanElement, IProps>(
 				})}
 				ref={ref}
 			>
+				{contentBefore && (
+					<span className="label-item label-item-before">
+						{contentBefore}
+					</span>
+				)}
```

Comment:
I think we could get creative with this and add the ability to add more than one element with `label-item-before`. First we will need to extract this into another component, `<ClayLabel.ItemBefore>` and that can just be in this same file. Then we probably want to have some logic to handle multiple items so that both the examples work below when passed to `contentBefore`

```jsx
// Handles all non React.Fragment items by wrapping them with `<ClayLabel.ItemBefore />` internally
<ClayIcon />


// If user passes in a React.Fragment we expect every child to be <ClayLabel.ItemBefore />
<>
	<ClayLabel.ItemBefore />
	<ClayLabel.ItemBefore />
</>
```

I think you would set the type for contentBefore to be 
```ts
contentBefore?: React.ReactFragment | typeof ItemBefore;
```

And I think  the rendering logic would be something like

```jsx
{
	contentBefore && contentBefore.type === React.Fragment ? (
		contentBefore
	) : (
		<ClayLabel.ItemBefore>{children}</ClayLabel.ItemBefore>
	);
}
```